eng/config/PublishData.json

```diff
@@ -27,6 +27,15 @@
           "vsBranch": "rel/d16.1",
           "vsMajorVersion": 16
         },
+        "dev16.1-preview3-vs-deps": {
+            "nugetKind": ["Shipping", "NonShipping"],
+            "version": "3.1.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev16.1", "dev16.1p2", "dev16.1p3" ],
```

Comment:
?

Reply:
Why not `"channels": [ "dev16.1", "dev16.1p3" ]`